src/main/webapp/i18n/en/textExercise.json

```diff
@@ -29,7 +29,17 @@
             "checkPlagiarism": "Check Plagiarism",
             "exampleSubmissionsRequireExercise": "To create an example submission, you first need to finish creating the text exercise. You will be redirected automatically.",
             "saveSuccessful": "Your answer was saved successfully!",
-            "error": "An error occurred. Please try again later."
+            "error": "An error occurred. Please try again later.",
+            "tutorEffortStatistics": {
+                "title": "Tutor Effort Statistics",
+                "numberOfSubmissionsAssessed": "Number of total submissions involved",
+                "totalTimeSpent": "Total time spent grading (Minutes)",
+                "averageTime": "Grading Pace (Submissions/Min)",
```

Comment:
Usually Speed is the one which has time in the denominator of a fraction.
Pace would be vice versa 👼